buildSrc/src/main/java/Dependencies.kt

```diff
@@ -45,6 +46,7 @@ object Dependencies {
     const val testing_junit = "junit:junit:${Versions.junit}"
     const val testing_robolectric = "org.robolectric:robolectric:${Versions.robolectric}"
     const val testing_mockito = "org.mockito:mockito-core:${Versions.mockito}"
+    const val testing_mockito_kotlin = "com.nhaarman.mockitokotlin2:mockito-kotlin:${Versions.mockito_kotlin}"
```

Comment:
We didn't add this dependency on purpose so far. But we added some helpers to `support-test`. What do you need it for here? :)

Reply:
That's fair. I took the bits that I needed from it and added it to `support-test`. Mockito's argument matcher captures a null at first so it fails unless you do some casting.